layouts/partials/footer.html

```diff
@@ -1,52 +1,44 @@
 {{ $links := .Site.Params.links }}
-<footer class="d-print-none">
-  <div class="footer__links">
-    <nav>
-      {{ with site.GetPage "page" "docs/tutorials/stateless-application/hello-minikube" }}<a href="{{ .RelPermalink }}">{{ .LinkTitle }}</a>{{ end }}
-      {{ $sections := slice "docs/home" "blog" "training" "partners" "community" "case-studies" }}
-      {{ range $sections }}
-      {{ with site.GetPage "section" . }}<a class="text-white" href="{{ .RelPermalink }}">{{ .LinkTitle }}</a>{{ end }}
-      {{ end }}
-    </nav>
-  </div>
-  <div class="container-fluid">
+<footer class="bg-dark py-5 row d-print-none">
+  <div class="container-fluid mx-sm-5">
```

Comment:
It looks like the `row` class is adding extra margins to the footer, causing the overflow and scroll issue mentioned in review comment https://github.com/kubernetes/website/pull/47789#pullrequestreview-2284394122.
Below is a suggestion that worked based on my testing using the browser's developer tools.

```suggestion
<footer class="bg-dark py-5 d-print-none">
  <div class="container-fluid">
```

Reply:
Might be the wrong fix, though. `col` class should only be used as a child of something with `row`, I think.

Reply:
I thought that the `col` class elements were already nested inside a `row` class div. I haven't had a chance to extensively test this layout.

https://github.com/kubernetes/website/blob/ff933be420f1e86f006a1c6a3599dd7ea392c4e0/layouts/partials/footer.html#L2-L4